server/homebrew.api.js

```diff
@@ -60,6 +57,14 @@ const getBrew = (accessType, stubOnly = false)=>{
 			// Combine the Homebrewery stub with the google brew, or if the stub doesn't exist just use the google brew
 			stub = stub ? _.assign({ ...excludeStubProps(stub), stubbed: true }, excludeGoogleProps(googleBrew)) : googleBrew;
 		}
+		const authorsExist = stub?.authors?.length > 0;
+		const isAuthor = stub?.authors?.includes(req.account?.username);
+		const isInvited = stub?.invitedAuthors?.includes(req.account?.username);
+		if(accessType === 'edit' && (authorsExist && (!isAuthor || !isInvited))) {
```

Comment:
This is causing another issue; now nobody can access any brews because if you are an author, you are not invited, and if you are invited, you are not an author. That is, `(!isAuthor || !isInvited)` is always true.

```suggestion
		if(accessType === 'edit' && (authorsExist && !(isAuthor || isInvited))) {
```

Reply:
🤦 thank you